src/eva/vision/models/networks/backbones/pathology/others.py

```diff
@@ -0,0 +1,33 @@
+"""Pathology FMs from other/mixed entities."""
```

Comment:
why `others.py` and not its own file like the other models? (gigapath.py would be easier to find)

Reply:
The names of the other files in that dir usually correspond to the entity that produced the model (e.g. kaiko, owkin). For gigapath there isn't a single entity so I couldn't follow that same naming convention. But still, I think you're right that it'd be better to name it `gigapath.py`, I will update this.

Reply:
Done